### PR TITLE
Install create react app and named project tuiter

### PR DESCRIPTION
**Pull request by Zevez7**
Require 2 reviewer for this first create react app project

### CODE REVIEW

**Comment by hzhou74** [comment]
Review your newly created.

File: tuiter/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```
Comment (hzhou74) [inline tuiter/.gitignore:12]: good job on put all those in gitignore

Comment (hzhou74) [reply tuiter/.gitignore:12]: should we share the build so when we download there are no version conflict?

Comment (Zevez7) [reply tuiter/.gitignore:12]: The main branch is public so anyone can download it. So it is shared. To create branches from the main you need to become a collaborator. So once they become collaborator they can get main branch and make their own branch to change and send a pull request to have their change push to main.

Comment (Zevez7) [reply tuiter/.gitignore:12]: you should pull from main branch so you have the latest version of the main branch.

Comment (Zevez7) [reply tuiter/.gitignore:12]: oh wait. the build file isn't needed since you can run build anytime you want without the build file.

**Comment by thematthewchan** [comment]
Looks good to me

**Comment by Zevez7** [comment]
accepting pull request after 2 reviwer

**Pull request merged**